kai-client-ci/files.py

```diff
@@ -0,0 +1,96 @@
+import json
+import platform
+import shutil
+import stat
+import sys
+import time
+import zipfile
+from datetime import datetime
+import logging
+import os
+import pandas as pd
+
+import git
+
+import requests
+from dotenv import load_dotenv
+
+from logger import get_logger
+
+logger = get_logger(__name__)
+
+
+def download_file(url, file_path):
+    response = requests.get(url, stream=True)
+    if response.status_code == 200:
+        with open(file_path, 'wb') as file:
+            for chunk in response.iter_content(chunk_size=8192):
+                file.write(chunk)
+        logger.info(f'Downloaded {file_path}')
+    else:
+        logger.error(f'Failed to download the file {file_path} | Status code: {response.status_code}')
+
+
+def rename_extracted_folder(base_folder, target_folder_name):
+    '''
+        Renames the extracted source code folder which is in format konveyor-kai-[uid] to a standard name 'kai'
+    '''
+    extracted_folders = [
+        folder for folder in os.listdir(base_folder)
+        if os.path.isdir(os.path.join(base_folder, folder)) and folder.startswith('konveyor-kai-')
+    ]
+
+    original_folder_path = os.path.join(base_folder, extracted_folders[0])
+    target_folder_path = os.path.join(base_folder, target_folder_name)
+
+    try:
+        shutil.move(original_folder_path, target_folder_path)
+    except Exception as e:
+        logger.error(f'Failed to rename folder: {e}')
+
+
+def clone_repository(app_name, repository_url, branch):
+    clone_dir = os.path.join('data', f'{app_name}')
+    git.Repo.clone_from(repository_url, clone_dir, branch=branch)
+    logger.info(f"Repository {app_name} cloned into 'data' with branch 'main'")
```

Comment:
Let it print the branch name as well .

logger.info(f"Repository {app_name} cloned into 'data' with branch {branch}")

Reply:
fixed in ea9c62e2a09ae0aebebbb67cd9c2e03408e55493